### ISSUE TITLE
Error summary producing empty anchor with href

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#2650]
This is an accessibility issue reported by Total Validator:

```
[WCAG2 1.1.1/2.4.4/2.4.9 (A/AAA)] Link text is missing:
<a name="fr-error" href="#">
```

Removing the `href` attribute fixes this problem. However, our thinking is that having a single name is a problem when both the top and bottom error summaries are shown. It is also a problem in a portal environment.

So, since as far as we know, we are never pointing to the error summary, and this is not documented, we've decided to completely remove this anchor.
